README.md

```diff
@@ -18,15 +18,14 @@ This set of instructions will create your own copy of this boilerplate directory
 3. Set up the two following variables and execute the `bootstrap.sh` script to initialise the git repository and replace all references to `openfisca_country_template` with references to your new country package in the code base:
 
 ```sh
-export COUNTRY_NAME=France  # set the name of your country here; you should keep all capitals, and replace any spaces in the name by underscores
-export REPOSITORY_URL=https://github.com/YOUR_ORGANISATION/OpenFisca-$COUNTRY_NAME  # set here the URL of the repository you created in step 2.
 ./bootstrap.sh
 ```
+(Note: you may need to first run `chmod +x ./bootstrap.sh` to grant execute permission)
 
 4. Push your changes to your git host:
 
 ```sh
-git push origin master
+git push origin main
```

Comment:
> I don't see how that can work when the URL we set the remote to is HTTPS instead of SSH… But that can be out of the scope of this PR since the issue is already present in the current code.

Reply:
The latest commits exclude adding the remote via bootstrap and instead instruct them to do it. This allows them the ability to set the remote whichever way they are comfortable and solves this issue